<?xml version="1.0"?>
<launch>
<param name="/use_sim_time"     value="True"/>
<arg name="gui"                 default="true"/>		
<arg name="debug_visualization" default="false"/>
<arg name="ns"                  default="robot_1"/>
	
<rosparam file="$(find map_change_detection)/param/parameters.yaml" ns="$(arg ns)"/>

<!--Map change detection -->

<node pkg="map_change_detection" name="map_updater" type="map_change_detection_node" ns="$(arg ns)">
    <param name="visualize_on_rviz"     value="$(arg gui)"/>
    <param name="visualize_debug_stuff" value="$(arg debug_visualization)"/>
    <param name="base_frame"            value="$(arg ns)/base_link" />
    <param name="scan_topic"            value="/$(arg ns)/laser_scan" />
    <param name="odom_topic"            value="/$(arg ns)/odom" />
</node>


<!-- Map updater -->

<!-- <node pkg="map_change_detection" type="map_change_detection_processer_node" name="updated_map_processer" output="screen" ns="$(arg ns)">
    <param name="map_topic"             value="/map" />
    <param name="processed_map_topic"   value="/$(arg ns)/processed_map" />
    <param name="changed_cells_topic"   value="/$(arg ns)/map_updater/changed_cells" />
</node> -->


</launch>
